admin/src/graphql/listAllContributions.js

```diff
@@ -0,0 +1,34 @@
+import gql from 'graphql-tag'
+
+export const listAllContributions = gql`
+  query (
+    $currentPage: Int = 1
+    $pageSize: Int = 25
+    $order: Order = DESC
+    $statusFilter: [ContributionStatus!]
+  ) {
+    listAllContributions(
+      currentPage: $currentPage
+      pageSize: $pageSize
+      order: $order
+      statusFilter: $statusFilter
+    ) {
+      contributionCount
+      contributionList {
+        id
+        firstName
+        lastName
+        amount
+        memo
+        createdAt
+        contributionDate
+        confirmedAt
+        confirmedBy
+        state
```

Comment:
Could this be status? In the database it is called status, so it would be nice, not to change the name

Reply:
we will have an extra adminListAllContribution in the near future ... there you could adjust this.

Reply:
I have created an issues  https://github.com/gradido/gradido/issues/2671